workflows/templates/inspection-build-template.yaml

```diff
@@ -0,0 +1,164 @@
+# Inspection Build Workflow Template
+#
+# This workflow builds an inspection image from the given Specification
+# Additionally, it mounts an artifact repository to store the resulting artifacts.
+#
+# The ouputs are stored in s3 as such:
+# s3://<bucket>/<prefix>/<workflow-name>/<workflow-step>
+#
+# NOTE: The artifact repository has to be configured via ConfigMap, see
+# https://github.com/argoproj/argo/blob/master/ARTIFACT_REPO.md
+# or use the Argo ansible role: https://github.com/CermakM/ansible-role-argo-workflows
+#
+# successCondition: `n` pods have finished successfully and are present in the cluster
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
```

Comment:
shall we have a 'thoth-station.ninja/template-version' annotation here too?

Reply:
I can add it there, but these are not instantiated.